websites/N/Notespace/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+	clientId: "1193651901221306378",
+});
+
+const enum Assets {
+	icon = "icon",
```

Comment:
We prefer image URLs rather than Discord asset names in this repository. 
Asset names should also be capitalized.
```suggestion
	Icon = "https://i.imgur.com/W7Ybot8.png",
```

Reply:
alright ill wil be changing that